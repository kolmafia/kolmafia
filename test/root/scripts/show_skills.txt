test skill add 12
test skill add 121
test skill add 152
skills
skills passive
test skill remove 12
test skill remove 121
test skill remove 152
skills
