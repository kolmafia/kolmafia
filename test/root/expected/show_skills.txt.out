
Torso Awareness (0 mp)
Summon Holiday Fun! (11 mp)
Communism! (0 mp)

Torso Awareness (0 mp)

